practice-two/src/js/views/generateFormProductHtml.js

```diff
@@ -0,0 +1,68 @@
+const productFormTemplate = (product = {}) => {
+  const {
+    id = '',
+    name = '',
+    category = '',
+    sku = '',
+    quantity = '',
+    cost = '',
+    price = '',
+    status = '',
+  } = product;
+  const isEditForm = Object.keys(product).length > 0;
+
+  const formProductHTML = `
+    <div class="modal-overlay">
+      <form action="javascript:void(0)" method="post" class="form-container">
+        <h2 class="add-product-title">${isEditForm ? 'Edit Product' : 'Add Product'}</h2>
+        <div class="form-group">
+          <label for="name">Product Name:</label>
+          <input id="name" data-field-name="Name" name="name" value="${name}" placeholder="Enter product name" class="form-input">
+          <p class="error-msg" data-field-error="Name"></p>
+          </div>
+        <div class="form-group">
+          <label for="category">Category:</label>
+          <select id="category" data-field-name="category" class="form-input">
+            <option ${category === 'Skin care' && 'selected'}>Skin care</option>
+            <option ${category === 'Face care' && 'selected'}>Face care</option>
+            <option ${category === 'Lips care' && 'selected'}>Lips care</option>
```

Comment:
Duplicated code here. Can you define list of category and loop through it for the options